egs/swbd/s5/local/rnnlm/run_lstm_d.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
```

Comment:
I think you should introduce the 'tuning' directory concept and introduce soft links to the best current model.  Otherwise this won't scale well.